content/en/opentelemetry/otlp_endpoint.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Datadog OTLP Trace Intake Endpoint
+private: true
+further_reading:
+  - link: "https://opentelemetry.io/docs/concepts/sdk-configuration/general-sdk-configuration/"
+    tag: "External"
+    text: "General OpenTelemetry SDK Configuration"
+  - link: "https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/"
+    tag: "External"
+    text: "OpenTelemetry Environment Variable Spec"
+  - link: "https://opentelemetry.io/docs/reference/specification/protocol/exporter/ "
+    tag: "External"
+    text: "OpenTelemetry Protocol Exporter"
+---
+{{< site-region region="ap1,gov" >}}
+<div class="alert alert-danger">This feature is not available for the {{< region-param key=dd_datacenter code="true" >}} site.</div>
+{{< /site-region >}}
+
+## Overview
+
+Datadog's OpenTelemetry protocol (OTLP) intake API endpoint allows you to send traces directly to Datadog, without running the [Datadog Agent][2] or [OpenTelemetry Collector + Datadog Exporter][1].
+
+You might prefer this option if you don't want the overhead of using the Datadog Agent or OpenTelemetry Collector.
+
+## Configuration
+
+To export OTLP data to the Datadog OTLP intake endpoint:
+
+1. [Configure the OTLP HTTP Protobuf exporter](#configure-the-exporter).
+   - Set the Datadog OTLP intake endpoint.
+   - Configure the required HTTP headers.
+1. (Optional) [Set `dd-otel-span-mapping` HTTP header](#optional-map-or-filter-span-names) to map or filter spans.
+
+### Configure the exporter
+
+To send OTLP data to the Datadog OTLP intake endpoint, you need to use the OTLP HTTP Protobuf exporter.
+
+#### Automatic instrumentation
+
+If you are using [OpenTelemetry automatic instrumentation][3], set the following environment variables:
+
+```shell
+export OTEL_EXPORTER_OTLP_TRACES_PROTOCOL="http/protobuf"
+export OTEL_EXPORTER_OTLP_TRACES_ENDPOINT="{{< region-param key="otlp_trace_endpoint" >}}"
+export OTEL_EXPORTER_OTLP_TRACES_HEADERS="dd-protocol=otlp,dd-api-key=${DD_API_KEY},dd-otlp-source=${YOUR_ORG}"
+```
+
+#### Manual instrumentation
+
+If you are using manual instrumentation with OpenTelemetry SDKs, configure the OTLP HTTP Protobuf exporter programmatically.
+
+<div class="alert alert-info">Replace <code>${YOUR_ENDPOINT}</code> with {{< region-param key="otlp_trace_endpoint" code="true" >}}. This is based on your <a href="/getting_started/site/">Datadog site</a>, which is {{< region-param key=dd_datacenter code="true" >}}.</div>
```

Comment:
I know we've already told them this feature is unavailable for US-FED and AP1, if you select one of those, `{{< region-param key="otlp_trace_endpoint" code="true" >}}` renders as `undefined`, which is a little awkward. Can you define it to say `not supported` or something instead?

Reply:
Other than this, LGTM, so I will approve, but it would still be nice to address this :)